src/main/java/com/mosaic/Weatherforecast.java

```diff
@@ -0,0 +1,40 @@
+package com.mosaic;
+
+import java.time.LocalDateTime;
+
+public class Weatherforecast {
+    private LocalDateTime time;
+    private float precipitationIntensity;
+    private int percipitationProbability;
+    private int percipitationType;
```

Comment:
Interesting is that here the type differs from one introduced inside Apiresponse